app/controllers/pages_controller.rb

```diff
@@ -4,14 +4,20 @@ class PagesController < ApplicationController
 
   def home
     @people = Person.all
-    @active_people = Person.joins(:bookings).merge(Booking.active)
+    @active_people = Person.active
 
     @bookings = Booking.all
     @active_bookings = @bookings.active
     @weekly_bookings = @bookings.last_week
 
     @active_charges = Charge.joins(:booking).merge(Booking.active)
 
+    @target_bond_people = @active_people.select do |p|
+      p.active_booking.bondable? && p.active_booking.bond_total <= 500
```

Comment:
Constant for now sounds good, with option to set as ENV var if we get feedback in that vein from users. 

I like the idea of a future slider if we have time; I think this aligns well with filters & etc that we've been discussing.